_posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md

```diff
@@ -74,21 +75,25 @@ SPPushNotification *event = [SPPushNotification build:^(id<SPPushNotificationBui
 [tracker trackPushNotificationEvent:event];
 {% endhighlight %}
 
+MIKE TO ADD DOCUMENTATION LINK.
+
 <h2><a name="swift-demo">3. Swift demonstration app</a></h2>
 
 To provide an example of how the tracker integrates into a Swift project, we've added a Swift demonstration app.
 
 Helpful points are addressed: [exception catching, and importing][swift-docs].
 
-Also take note of [how to view generated interfaces][swift-interfaces] which are essential for finding the type signatures of the methods translated from Objective-C to Swift.
+I DON'T UNDERSTAND THE ABOVE SENTENCE ^^
+
+It's also important to familiarize yourself with [how to view generated interfaces][swift-interfaces] - essential for finding the type signatures of the tracker methods when translated from Objective-C to Swift.
 
 <h2><a name="xcode9">4. XCode 9 bugfixes</a></h2>
 
-To improve compatibility with XCode 9 we have updated the Tracker as follows:
+To improve compatibility with XCode 9 we have fixed a naming conflict in the SnowplowTests bundle ([issue #343][343]).
 
-* Fix a naming conflict in the SnowplowTests bundle ([#343][343])
+We also fixed a compile-time error found in the [Obj-C Client for Iglu][iglu-client], and updated a dependency of the Tracker.
```

Comment:
We also fixed a compile-time error found in both the Tracker and one of its dependencies, the [Obj-C Client for Iglu][iglu-client].

Reply:
What's the ticket for this fix @mhadam ?

Reply:
https://github.com/snowplow/iglu-objc-client/issues/7